gpt_index/indices/response/response_builder.py

```diff
@@ -7,65 +7,39 @@
 2) create and refine separately over each chunk, 3) tree summarization.
```

Comment:
i'd add the top-level classes to the __init__ file so that users can just import via `from gpt_index.indices.response import ...`